### PR TITLE
Add README and supported python versions to PyPI description

### LINKED ISSUE #256: Update PyPI page
It would be nice if the `README.rst` were included in the `setup.py` `long_description` in addition to the `CHANGES.rst` so that users browsing PyPI could read the README without having to travel to GitHub.

Would also be nice if the trove classifiers in `setup.py` reflected which versions of Python were officially supported (e.g. `'Programming Language :: Python :: 3.5'`).

### PR DESCRIPTION
**Pull request by johnthagen**
Fix #256.

### CODE REVIEW

**Comment by johnthagen** [comment]
Per #259, updated operating system classifiers so users are aware Windows is not currently supported.

File: setup.py [hunk 1]
```diff
@@ -5,9 +5,11 @@
 
 from setuptools import setup
 
+with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as fp:
+  LONG_DESCRIPTION = fp.read() + '\n'
```
Comment (kwlzn) [inline setup.py]: here's a preview of what this concatenation would look like: https://gist.github.com/kwlzn/97f740834edcc03b6e729a3b18d641dc

note the disjoint styling between the concatenation. seems to me that we'd probably want to change the style of the CHANGES.rst header, which is currently:

```
=======
CHANGES
=======
```

to something like:

```
Release Notes
===========
```

to make this line up nicely.

**Comment by johnthagen** [comment]
@kwlzn Good catch, updated PR to appropriately.

**Comment by kwlzn** [comment]
LGTM!

**Comment by kwlzn** [comment]
merged! thanks @johnthagen!

**Pull request merged**